### PR TITLE
Added world changing python code in python/hello.py

### PR DESCRIPTION
**Pull request by simucollab**
Ever since  the 1960s I wanted to print 'Hello Worlds!' in an awesome language and my dream has come true.

Please let me fix this for myself by adding this complicated patch to your repository.

Hopefully fixes https://github.com/richardskumat/pubcollab/issues/5 and let's us release v3 of our glorious "app".

### CODE REVIEW

**Comment by richardskumat** [comment]
Amazing work!

File: python/hello.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+def hello(world):
```
Comment (richardskumat) [inline python/hello.py]: That's a really creative way to create python functions.

File: python/hello.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    print('Hi ' + world)
```
Comment (richardskumat) [inline python/hello.py]: So it prints Hi plus a variable passed with

hello('variable').

File: python/hello.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+hello('Richard')
```
Comment (richardskumat) [inline python/hello.py]: Looks good.

File: python/hello.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+hello('Kate')
```
Comment (richardskumat) [inline python/hello.py]: Need to test this multiple times, just to be sure.

File: python/hello.py (hunk 1 above)
Comment (simucollab) [inline python/hello.py]: Thanks! I worked hard to achieve that.

File: python/hello.py (hunk 2 above)
Comment (simucollab) [inline python/hello.py]: Indeed it does.

File: python/hello.py (hunk 4 above)
Comment (simucollab) [inline python/hello.py]: Indeed. I've spent hours on testing this code and "It just works!".

**Comment by richardskumat** [comment]
Agreed. Wonderful code, merged.

**Comment by richardskumat** [comment]
Avoid bugs by automatically running your tests.

Continuous integration can help catch bugs by running your tests automatically.
Merge your code with confidence using one of our continuous integration providers.

Thanks github.

**Pull request merged**